src/source/ads.rst

```diff
@@ -395,6 +396,17 @@ per customer. Can either be set when creating an ad or when updating an
 existing ad. However, once set, it can no longer be modified. When fetching an
 existing ad which does not have a **vendorId**, the field is omitted.
 
+.. index:: microTip
+.. _ad_microTip:
+
+microTip
+""""""""
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature used in the context of SMB bundles (currently only by Marktplaats tenant),
```

Comment:
our sellside customers don't know the concept of SMB or the word 'bundles'; Please reword using information about packages & features.

Reply:
hm, I thought "bundles" was even in the link of the packaging..do you happen to have that link, I lost it together with the lingo we are using in it

Reply:
found it, nevermind

Reply:
fixed